frontend/src/k8sTypes.ts

```diff
@@ -22,6 +22,7 @@ export enum KnownLabels {
   PROJECT_SHARING = 'opendatahub.io/project-sharing',
   MODEL_SERVING_PROJECT = 'modelmesh-enabled',
   DATA_CONNECTION_AWS = 'opendatahub.io/managed',
+  MODEL_REGISTRY_SHARING = 'opendatahub.io/model-registry-sharing',
```

Comment:
You're not actually applying this label to your new roleBindings I think. We probably don't need it now that we're filtering by `app.kubernetes.io/name` in `ModelRegistriesManagePermissions` and using the `component=model-registry` labelSelector in `useRoleBindings`.

Reply:
does the "probably" mean we should get confirmation from @lucferbux ?

Reply:
I think we can just ditch it, it was my idea and turns out I was wrong about it being necessary